src/app/login/login.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { LoginPageComponent } from 'login-page/login-page.component';
+
+@NgModule({
+   declarations: [
+      LoginPageComponent
+   ],
+   imports: [
+      CommonModule
+   ],
+   exports: [
+      LoginPageComponent
+   ]
+})
+export class LoginModule { }
```

Comment:
I'm not sure, that you need module for one page.

Reply:
In the requirements for this task I've read: "2. Create markup, module and component for login page."